### PR TITLE
Changed the live-coding sample to use the live share sdk nodejs

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/meetings-live-code-interview/nodejs/appPackage/manifest.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "accentColor": "#5558AF",
   "configurableTabs": [
     {
-      "configurationUrl": "https://{{Domain Name}}/configure",
+      "configurationUrl": "https://c867-116-75-13-29.ngrok.io/config",
```
Comment (ChetanSharma-msft) [inline samples/meetings-live-code-interview/nodejs/appPackage/manifest.json:28]: Can't we add placeholder for domain name?

File: samples/meetings-live-code-interview/nodejs/ClientApp/public/index.html [hunk 2]
```diff
@@ -2,12 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Website created using create-react-app"
+      content="Web site created using create-react-app"
```
Comment (ChetanSharma-msft) [inline samples/meetings-live-code-interview/nodejs/ClientApp/public/index.html:10]: We can change the content as per the project title.

Comment (ChetanSharma-msft) [inline samples/meetings-live-code-interview/nodejs/ClientApp/public/index.html:10]: We can change the content as per the project title.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
One cosmetic comment.

**Review by Wajeed-msft — APPROVED** [review]
Could you please mention Live Share SDK in ReadMe file? This would be similar to C# project.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**